noq_django/backend/models.py

```diff
@@ -348,4 +348,33 @@ def calculate_vat(self):
     
     def save(self, *args, **kwargs):
         self.calculate_vat()
-        super().save(*args, **kwargs)
\ No newline at end of file
+        super().save(*args, **kwargs)
+
+class SleepingSpace(models.Model):  # English class name
+    BED_TYPES = [
+        ('Dubbelsäng över/under', 'Dubbelsäng över/under'),
+        ('Singelsäng', 'Singelsäng'),
+        ('Madrass', 'Madrass'),
+    ]
+    
+    ROOM_LOCATIONS = [
```

Comment:
Question: Should we remove this and user the Product type instead for this information?